valhalla/mjolnir/admin.h

```diff
@@ -18,13 +18,18 @@
 using namespace valhalla::baldr;
 using namespace valhalla::midgard;
 
-namespace valhalla {
-namespace mjolnir {
+namespace valhalla::mjolnir {
+
+namespace bg = boost::geometry;
```

Comment:
I don't think we should change this to `valhalla::mjolnir`. I get that it's terser, but I'd rather continue with the same convention as in the rest of the project.

Reply:
not sure it makes a real difference but usually we keep those namespace aliases outside of our own namespaces